### PR TITLE
Bugfix sorting in project names separates upper and lowercase

### PR DESCRIPTION
**Pull request by rwrzesien**
Resolves https://github.com/Code-Poets/sheetstorm/issues/69

### CODE REVIEW

File: managers/factories.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import factory
+import factory.fuzzy
+
+from managers.models import Project
+
+
+class ProjectFactory(factory.DjangoModelFactory):
```
Comment (kbeker) [inline managers/factories.py]: Please add simple test for this factory

File: managers/views.py [hunk 2]
```diff
@@ -1,8 +1,9 @@
+from django.db.models.functions import Lower
```
Comment (kbeker) [inline managers/views.py]: This function just make string which is composed with Upper letter to lower letters?

File: requirements-development.txt [hunk 3]
```diff
@@ -10,6 +10,7 @@ django-allauth==0.37.1
 django-countries==5.3.2
 django-rest-auth==0.9.3
 djangorestframework==3.8.2
+factory-boy==2.11.1
```
Comment (kbeker) [inline requirements-development.txt]: If it is is normal requirements we don't have to add it also to requirements-develpment I think :)

**Comment by Karrp** [comment]
@rwrzesien Please add this pull requests to Sheet Storm project

File: managers/views.py (hunk 2 above)
Comment (rwrzesien) [reply managers/views.py]: Yes, but it does it on database side, during SQL query.

File: managers/factories.py (hunk 1 above)
Comment (rwrzesien) [reply managers/factories.py]: As discussed, we don't want to test factories.

File: requirements-development.txt (hunk 3 above)
Comment (rwrzesien) [reply requirements-development.txt]: Done :+1:

**Pull request merged**